src/build/app/mod.rs

```diff
@@ -930,22 +998,31 @@ impl<'help> App<'help> {
         self
     }
 
-    /// Allows adding a [``] alias, which function as "hidden" subcommands that
-    /// automatically dispatch as if this subcommand was used. This is more efficient, and easier
-    /// than creating multiple hidden subcommands as one only needs to check for the existence of
-    /// this command, and not all variants.
+    /// If this `App` instance is a subcommand, this method adds an alias, which
+    /// allows this subcommand to be accessed via *either* the original name, or
+    /// this given alias. This is more efficient, and easier than creating
+    /// multiple hidden subcommands as one only needs to check for the existence
+    /// of this command, and not all aliased variants.
+    ///
+    /// **NOTE:** Aliases defined with this method are *hidden* from the help
+    /// message. If looking for aliases that will be displayed in the help
```

Comment:
We don't use this style for docs

Reply:
Um, what's wrong? I believe the original sentence is grammatically incorrect.

Reply:
No. I am not sure what the right term for it is, but this is one of those styles of grammar that are used for instructions/documentations etc.. in technical writing.

Reply:
If you can't name it, than it's very likely nothing but a common mistake you happened to come across a lot. Documentation is often written by people who aren't into linguistics, not to mention that many of them aren't native speakers. And even if they are, there's a big difference between "I can speak English fluently" and "I can speak English _correctly_". This isn't English-specific: if you knew Russian and saw how Russian engineers write documentation, you would experience a very strong urge to burn it down along with authors - despite the fact that Russian is fairly forgiving when it comes to word order and such. 

I read a lot of English books these days (both technical and fiction, mostly fiction though), and I don't remember ever seeing this kind of clause. I did try to search for this thing before making a suggestion, to no avail. Anyway, I don't understand why you considered my suggestion a bad style, mind elaborate?

Reply:
We should be using "we" if we need a pronoun and we have been doing this in clap docs. But it sounds weird here which is why the sentence has been reduced to be a bit more direct succinct by eliminating pronoun.

I would just rather rephrase the whole sentence.

Reply:
I've been taught that English is very sensitive to pronoun elimination, and my whole experience tells me this particular kind of elimination is an error. I don't think pronounces in this kind of conditional clause are subject to elimination _at all_.

My acquaintance agreed that was an error and she told me that if I wanted to eliminate the pronoun here so desperately I could rewrite it as, "**When** looking for aliases that will be displayed in the help...", but it doesn't quite agree with the second part of the sentence. Also, there _are_ kinds of conditions where you do eliminate the pronoun, like, "Be kind. **Failing that** [if you're failing to be kind], bring evidence", but I don't think it works with "If", and I never saw such clauses longer than 4 words.

> We should be using "we" if we need a pronoun

This is not a mindless "always obey" kind of rule, common sense still applies. The sentence doesn't sound any better with "we" instead of "you". Arguably, it sounds worse.

I pretty much agree that the sentence better be rephrased and I welcome suggestions, but I'm still content with treating the original sentence as incorrect unless you can link to a reputable dictionary that allows it. Frankly, having conducted the research, at this point I'm almost convinced you're either projecting some rules from you're local region or company to international English or simply making this up.